Tests/SourceKitLSPTests/RenameTests.swift

```diff
@@ -1187,39 +1188,127 @@ final class RenameTests: XCTestCase {
   }
 
   func testAddLabelToEnumCase() async throws {
-    // We don't support renaming enum parameter labels at the moment
-    // (https://github.com/apple/sourcekit-lsp/issues/1228)
+    try await SkipUnless.sourcekitdCanRenameEnumCaseLabels()
     try await assertSingleFileRename(
       """
       enum MyEnum {
         case 1️⃣myCase(String)
       }
       """,
       newName: "newName(newLabel:)",
-      expectedPrepareRenamePlaceholder: "myCase",
+      expectedPrepareRenamePlaceholder: "myCase(_:)",
       expected: """
         enum MyEnum {
-          case newName(String)
+          case newName(newLabel: String)
         }
         """
     )
   }
 
-  func testRemoveLabelToEnumCase() async throws {
-    // We don't support renaming enum parameter labels at the moment
-    // (https://github.com/apple/sourcekit-lsp/issues/1228)
+  func testRemoveLabelFromEnumCase() async throws {
+    try await SkipUnless.sourcekitdCanRenameEnumCaseLabels()
     try await assertSingleFileRename(
       """
       enum MyEnum {
-        case 1️⃣myCase(label: String)
+        case myCase(label: String)
+      }
+
+      func test() {
+        _ = MyEnum.2️⃣myCase(label: "abc")
       }
       """,
       newName: "newName(_:)",
-      expectedPrepareRenamePlaceholder: "myCase",
+      expectedPrepareRenamePlaceholder: "myCase(label:)",
+      expected: """
+        enum MyEnum {
+          case newName(_ label: String)
```

Comment:
Should we be dropping the label completely here?

Reply:
I think it’s valuable to keep it as an internal label, similar to what we do for function completion (and yes, also easier to implement 🙈)